evm_loader/program/src/error.rs

```diff
@@ -0,0 +1,28 @@
+//! Error types
+
+use num_derive::FromPrimitive;
+use solana_program::{decode_error::DecodeError, program_error::ProgramError};
+use thiserror::Error;
+
+/// Errors that may be returned by the EVM Loader program.
+#[derive(Clone, Debug, Eq, Error, FromPrimitive, PartialEq)]
+pub enum EVMLoaderError {
```

Comment:
I would suggest renaming it to `EvmLoaderError`. In this case, it will be simple to use it (since this name confirmed rust snake case naming convention)